nomad_files/traefik-wall.hcl

```diff
@@ -0,0 +1,111 @@
+job "traefik-wall" {
+  region      = "global"
+  datacenters = ["dc1"]
+  type        = "service"
+  node_pool = "staging"
+
+  group "traefik" {
+    count = 2
+
+    network {
+      port "http" { }
+
+      dns {
+        servers = ["10.88.0.1", "128.112.129.209", "8.8.8.8", "8.8.4.4"]
+      }
+    }
+
+    service {
+      name = "traefik-wall-staging"
+      tags = ["node-${NOMAD_ALLOC_INDEX}"]
+      port = "http"
+
+      check {
+        name     = "alive"
+        type     = "tcp"
+        port     = "http"
+        interval = "10s"
+        timeout  = "2s"
+      }
+    }
+
+    task "traefik" {
+      driver = "podman"
+
+      config {
+        image        = "docker.io/library/traefik:v3.3"
+        network_mode = "host"
+
+        volumes = [
+          "local/traefik.toml:/etc/traefik/traefik.toml",
+          "local/dynamic.toml:/etc/traefik/dynamic.toml",
+        ]
+      }
+
+      template {
+        data = <<EOF
+defaultEntryPoints = ["http"]
+
+[log]
+  level = "DEBUG"
+
+[entryPoints]
+   [entryPoints.http]
+     address = ":{{ env "NOMAD_PORT_http" }}"
+   [entryPoints.http.forwardedHeaders]
+     insecure = true
+   [entryPoints.traefik]
+     address = ":9091"
+
+[api]
+  dashboard = false
+  insecure  = false
+
+[providers]
+  [providers.file]
+    filename = "/etc/traefik/dynamic.toml"
+[experimental.plugins.captcha-protect]
+modulename = "github.com/libops/captcha-protect"
+version = "v1.3.2"
+EOF
+
+        destination = "local/traefik.toml"
+      }
+
+      template {
```

Comment:
Everything in this template block is the "dynamic" configuration for Traefik. This is using the "file" provider - Traefik is usually configured via docker tags, but we can use a file instead.